Cargo.toml

```diff
@@ -20,6 +20,7 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs", "--html-in-header", "katex-header.html"]
 
 [dev-dependencies]
+bincode = "1.2.1"
```

Comment:
https://crates.io/crates/bincode
> Bincode attempts to protect against hostile data. There is a maximum size configuration available (`bincode::config::Bounded`), but not enabled in the default configuration. Enabling it causes pre-allocation size to be limited to prevent against memory exhaustion attacks.

We should probably enable `bincode::config::Bounded`.